worker/uniter/runner/jujuc/jujuctesting/relation.go

```diff
@@ -24,14 +24,17 @@ type Relation struct {
 	Units map[string]Settings
 	// UnitName is data for jujuc.ContextRelation.
 	UnitName string
+	// RemoteApplicationSettings is data for jujuc.ContextRelation
+	RemoteApplicationSettings Settings
 	// ApplicationSettings is data for jujuc.ContextRelation
-	ApplicationSettings Settings
```

Comment:
the comment needs to be updated. "LocalApplicationSettings"

Reply:
Fixed